packages/platform-android/src/commands/runAndroid/runOnAllDevices.js

```diff
@@ -39,6 +39,10 @@ function runOnAllDevices(
     const tasks = args.tasks || ['install' + toPascalCase(args.variant)];
     const gradleArgs = getTaskNames(args.appFolder, tasks);
 
+    if (args.port != null) {
+      gradleArgs.push('-PreactNativeDevServerPort=' + args.port);
```

Comment:
Quick q: Why this is called `Preact`?

Reply:
It's how you can pass parameters to Gradle, something like `-P reactNativeDevServerPort` should also work I guess? But I didn't bother to check it. cc @nhunzaker

Reply:
`-P` for "Set project property for the build script"

Directly from the gradlew usage stuff:

```
-P, --project-prop        Set project property for the build script (e.g. -Pmyprop=myvalue).
```

Having no space is definitely confusing.

Reply:
Ah, gotcha, thanks! I read it "Preact" and asked myself what's going on :joy: Thank you for explaining it!